packages/backend-core/src/middleware/passport/google.js

```diff
@@ -57,5 +59,19 @@ exports.strategyFactory = async function (config, callbackUrl, saveUserFn) {
     )
   }
 }
+
+exports.fetchStrategyConfig = async function (googleConfig) {
```

Comment:
I think we won't want to fallback to the environment variables here, as they will be set in the cloud for datasource integration for google sheets using our own google creds. I think that behaviour should only be used for the datasource integration and not for the sso process, so best to just read the config here and can probably remove this function

Reply:
Update incoming.